README.md

```diff
@@ -118,12 +118,13 @@ Socialize, eat together, music and honey time. (on toast)
 
 ### produce
 
-| Name  | Content                                       |
-| ----- | --------------------------------------------- |
-| id    | Unique ID for the fruit/vegetable             |
-| name  | Name of the fruit/vegetable                   |
-| type  | Type of the produce, i.e. fruit or vegetables |
-| image | Image of the fruit/vegetable                  |
+| Name   | Content                                                         |
+| ------ | --------------------------------------------------------------- |
+| id     | Unique ID for the fruit/vegetable                               |
+| name   | Name of the fruit/vegetable                                     |
+| type   | Type of the produce, i.e. fruit or vegetables                   |
+| image  | Image of the fruit/vegetable                                    |
+| season | An array of months that the produce is available in NZ (1 - 12) |
```

Comment:
You cant/shouldnt save an array in a databse

Reply:
Sorry, another out-of-date README entry. Season is intended to be stored as follows, using the `produce_available_months` table:

| Name       | Content                          |
| ---------- | -------------------------------- |
| ID         | Unique ID for the produce months |
| produce_id | ID referencing the produce table |
| month_id   | ID referencing the months table  |